manifests/kustomize/base/installs/generic/pipeline-install-config.yaml

```diff
@@ -11,6 +19,34 @@ data:
   cacheDb: cachedb
   pipelineDb: mlpipeline
   bucketName: mlpipeline
+
+  ## defaultPipelineRoot: Optional. Default pipeline root in v2 compatible mode.
+  ## https://www.kubeflow.org/docs/components/pipelines/sdk/v2/v2-compatibility/
+  ##
+  ## If the field is not set, kfp-launcher configmaps won't be created and 
+  ## v2 compatible mode defaults to minio://mlpipeline/v2/artifacts as pipeline
+  ## root.
+  ##
+  ## When not in Kubeflow Pipelines multi-user mode, the config works as you
+  ## would normally expect.
+  ##
+  ## In Kubeflow Pipelines multi-user mode, the config creates default
```

Comment:
Do all namespaces share same default pipeline root path as specified in `defaultPipelineRoot` field under multi-user mode? How do we control data access if all namespaces share same pipeline root?

Reply:
> Users can edit the  kfp-launcher configmap afterwards and their changes won't be overridden again by this config.

Users can configure different defaultPipelineRoot per namespace in `kfp-launcher` configmap. Do you have any suggestions to make this part easier to understand?

let me try to clarify in the PR first

Reply:
Thank you Yuan for the doc update! I understand it better now. I was originally thinking that how do we programmatically assign a pipeline root for each namespace? For example: gs://{$BUCKET_NAME}/v2/artifact/{$NAMESPACE}. But it seems to be out-of-scope for current version, because we need to edit kfp-launcher-configmap.yaml in each user's namespace. 

Also understand it is hard to configure pipeline root based on namespace parameter, the current solution SGTM.